tools/gcp_init/installer.py

```diff
@@ -0,0 +1,713 @@
+
+    log.info('upgrading local pip')
+    fab.local('sudo pip install --upgrade pip')
+
+    log.info('installing local wheel')
+    fab.local('sudo pip install wheel')
+
+
+def RemoteOsSetup(host=None):
+  """Setup up OS on remote host for turbinia.
+
+  Args:
+    host (string): hostname for remote host
+  """
+  if host:
+    log.info('performing operations on host: {0:s}'.format(host))
+    with fab.hide('running'):
+      with fab.settings(host_string=host,
+                        output_prefix=False,
+                        timeout=3,
+                        connection_attempts=60):
+
+        log.info('updating remote apt sources.list')
+        fab.sudo('perl -pi -e \x27s/^(deb.+main)$/$1 contrib non-free/\x27 '
+                 '/etc/apt/sources.list')
+
+        log.info('installing remote os updates')
+        fab.sudo('apt-get -y update; apt-get -fy dist-upgrade')
+
+        log.info('installing remote dependencies')
+        fab.sudo('apt-get -fy install git liblzma-dev sudo '
+                 'cpio pax rsync socat pv vim-nox wget curl '
+                 'tmux screen tcpdump '
+                 'python-pip python-wheel python-virtualenv '
+                 'python-openssl ')
+
+        log.info('upgrading remote pip')
+        fab.sudo('pip install --upgrade pip')
+        log.info('installing remote wheel')
+        fab.sudo('pip install wheel')
+
+        log.info('adding turbinia user')
+        fab.sudo('grep -q turbinia /etc/group || '
+                 'groupadd -g {0:d} turbinia'.format(tcfg.TURBINIA_GID))
+        fab.sudo(('grep -q turbinia /etc/passwd || '
+                  'useradd -u {0:d} -g {1:d} -G disk -c turbinia '
+                  '-s /bin/bash -m -d /home/turbinia '
+                  'turbinia').format(tcfg.TURBINIA_UID, tcfg.TURBINIA_GID))
+        fab.sudo('passwd -l turbinia')
+        fab.sudo('grep -q turbinia /etc/sudoers || '
+                 'echo "turbinia ALL=(ALL:ALL) NOPASSWD: ALL" >> /etc/sudoers')
+
+
+def RemoteTurbiniaSetup(opts, host=None):
+  """Setup up remote host for turbinia.
+
+  Args:
+    opts (dict): cli options data structure
+    host (string): hostname for remote host
+  """
+  remote_cache = opts.cache
+  date_suffix = time.strftime('%Y%m%d_%H%M%S')
+  if host:
+    log.info('performing operations on host: {0:s}'.format(host))
+    with fab.hide('running'):
+      with fab.settings(host_string=host,
+                        output_prefix=False,
+                        timeout=3,
+                        connection_attempts=60):
+        log.info('installing remote plaso')
+        fab.sudo(('sudo pip install --use-wheel --no-index --find-links={0:s} '
+                  '-r {0:s}/plaso-reqts.txt plaso').format(remote_cache))
+
+        log.info('updating turbinia directory')
+        fab.sudo('mv {0:s}/turbinia {1:s}-{2:s}'.format(opts.tmpdir,
+                                                        opts.tdir, date_suffix))
```

Comment:
The default tdir is /usr/local/turbinia. If it was overriden by a different path, I think the path should be added to the config file, which should help us avoid us hardcoding /usr/local/turbinia/turbiniactl in the systemd script. Any thought?